### ISSUE TITLE
Class function to calculate cohort based on first paediatric assessment date

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/rcpch-audit-engine#14]
```
    #TODO Class function to calculate cohort based on first paediatric assessment date
```

https://github.com/rcpch/rcpch-audit-engine/blob/4cafd1c7bdc158daf59f164c65a5e6317529099e/rcpch-audit-engine/epilepsy12/models.py#L357

**Comment by eatyourpeas** [comment]
This has been implemented in practice with commit a414ba4
Need to finalise what definition @coldunk wants for cohorts - ie what are the month cut offs

**Comment by eatyourpeas** [comment]
This was reinstated some time back and remains live in the codebase
